packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py

```diff
@@ -47,6 +59,13 @@ class NodePatch(InputSchemaWithoutCamelCase):
         str, Any
     ] | None = None  # NOTE: it is used by frontend for File Picker
 
+    def to_model(self) -> PartialNode:
```

Comment:
we call it now `to_domain_model`  so that we are more explicit

Reply:
I agree